daofund-score/src/main/java/com/iconbet/score/daofund/DaoFund.java

```diff
@@ -0,0 +1,303 @@
+package com.iconbet.score.daofund;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+import score.Address;
+import score.ArrayDB;
+import score.Context;
+import score.DictDB;
+import score.VarDB;
+import score.annotation.EventLog;
+import score.annotation.External;
+import score.annotation.Payable;
+
+public class DaoFund {
+	
+	public String TAG = "ICONbet DAOfund";
+	
+    private static final String ADMINS = "admins";
+    private static final String WITHDRAW_COUNT = "withdraw_count";
+    private static final String WITHDRAW_RECORD = "withdraw_record";
+    private static final BigInteger X_6 = new BigInteger("1000000"); // 10 ** 6
+    private static final BigInteger BATCH_SIZE = BigInteger.valueOf(100L);
+    
+
+	private final ArrayDB<Address> admins = Context.newArrayDB(ADMINS, Address.class);
+	private final VarDB<BigInteger> withdraw_count = Context.newVarDB(WITHDRAW_COUNT, BigInteger.class);
+	private final DictDB<String,LinkedList> withdraw_record = Context.newDictDB(WITHDRAW_RECORD, LinkedList.class);
```

Comment:
probably we will need to use BranchDB
https://github.com/cacomixtle/iconbet-java/blob/master/reward-distribution/src/main/java/com/iconbet/score/reward/RewardDistribution.java#L56

pretty similar the way it is accessed in py

self.withdraw_record[_withdraw_count]['withdraw_memo'] = _memo

py
variable[int index][String index] 

java
this._wagers.at(index).get(address);

https://github.com/cacomixtle/iconbet-java/blob/master/reward-distribution/src/main/java/com/iconbet/score/reward/RewardDistribution.java#L261

we will know once we deploy to dev :(

Reply:
Updated accordingly.